src/components/PageBlocks/CalendarBase.vue

```diff
@@ -258,10 +267,11 @@ export default {
       this.events = []
 
       this.options.feeds.forEach(feed => {
```

Comment:
We need to do Promise.all otherwise its not gonna work with multiple feeds.

Remove the finally's you added and instead do Promise.all(this.options.feeds.map()).finally
Also make sure to return the promises (273, 299). Reference another use of Promise.all to see how it works